cmd/setup.go

```diff
@@ -12,7 +12,9 @@ import (
 
 var setupCmd = &cobra.Command{
 	Use:   "setup",
-	Short: "setup holepunch",
+	Short: "Setup punch (run this first)",
```

Comment:
Should we have `punch --help` or `punch -h` suggest running `punch setup`?

Reply:
It prints out the `Short` from each command, so it is there.  @AndrewScibek, how do I make this more obvious to the user?
I guess I could put a longer comment in root.go?

Reply:
I added some basic getting started text in the most recent commit.  Check it out.